Sources/Authgear.swift

```diff
@@ -672,22 +715,49 @@ public class Authgear {
     public func application(
         _ application: UIApplication,
         continue userActivity: NSUserActivity,
-        restorationHandler: @escaping ([UIUserActivityRestoring]?) -> Void
+        restorationHandler: @escaping ([UIUserActivityRestoring]?) -> Void,
+        handler: VoidCompletionHandler? = nil
```

Comment:
I thought the functions' signature should be the same as the functions of `UISceneDelegate` and `UIApplicationDelegrate`. The callback function should be defined in the `AuthgearDelegate`

maybe something like?
```
public protocol AuthgearDelegate: AnyObject {
    func onLoginLinkVerificationFailed()
}
```

So we don't need to change the functions signature and only the function of `UIApplicationDelegrate` need to return bool.

Thought? @louischan-oursky 

We need to confirm the public API before implementing. Can you help list out the function signature in the spec or issue, so that we can discuss and confirm.